src/binder_base.c

```diff
@@ -143,8 +143,8 @@ binder_base_emit_queued_signals(
     /* Signal handlers may release references to this object */
     g_object_ref(self);
 
-    /* Emit the signals */
-    for (p = 0; self->queued_signals && p < BINDER_BASE_MAX_PROPERTIES; p++) {
+    /* Emit the signals, ignore the ANY property */
```

Comment:
Why

Reply:
Ah, I think I got it, never mind))

Reply:
The BINDER_BASE_PROPERTY_BIT would be doing 1 << -1 when using 0 as parameter which is obviously not correct.

Reply:
Yes, that's exactly what I realized)

Reply:
thanks